### ISSUE TITLE
Improve excuses user interface

### ISSUE THREAD

**Issue by nerdinand** [nerdinand/chorbasel-app#77]
From App feedback:

> Tolle App! Vielen Dank.
> 
> Wenn ich mich für eine Probe entschuldige, dann verschwindet der „entschuldigen“-Button. Ich fände es besser, wenn da stünde „entschuldigt“ und darunter ein gelber „Entschuldigung rückgängig machen“-Button wäre.

**Comment by hharen** [comment]
Do you agree with this solution?

**Comment by nerdinand** [comment]
No, not with this exact solution. But I haven't given it much thought yet from a UX perspective...

**Comment by hharen** [comment]
And from functional perspective?

**Comment by nerdinand** [comment]
Well, I do think it's necessary to be able to amend an excuse somehow. But I do think the history should still be visible, even if you change an excuse.

**Comment by hharen** [comment]
I see. So either show the actions in a history or have an active/not active state or something.

**Comment by nerdinand** [comment]
I really haven't thought much yet about the concept. I just trust it will fall into place when I fiddle with it. Maybe it will need a change to the data structure as well.
